TaskForTraining/TaskForTraining/Pages/HomePage.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Support.PageObjects;
+
+namespace TaskForTraining.Pages
+{
+    class HomePage
+    {
+        private IWebDriver driver;
+
+        public HomePage(IWebDriver driver)
+        {
+            this.driver = driver;
+            PageFactory.InitElements(driver, this);
+        }
+
+        public const string UName = "Test Auto";
+
+        [FindsBy(How = How.XPath, Using = "//span[@class='uname']")]
```

Comment:
Move your IWebElement fields to the top, they should be placed before constructor